### PR TITLE
User can view "detailed" version of item.

### PR DESCRIPTION
**Pull request by Stevenlee731**

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Hi Steve. Nice work, just delete all the commented code.

File: main.js [hunk 1]
```diff
@@ -100,50 +27,112 @@ function renderItem(item) {
   $itemDescription.classList.add('card-text')
   $learnMore.classList.add('card-text')
   $button.classList.add('btn', 'btn-primary')
-  $image.setAttribute('src', item.image)
+  $button.setAttribute('href', item.addToCart)
```
Comment (victormejia) [inline main.js]: clicking on the `Add To Cart` button will take me to Google's homepage, which is probably an unintended feature. I would suggest to leave this blank, and them implement this in a future feature.

Comment (Stevenlee731) [reply main.js]: Thanks @victormejia. Removed the google link!

**Review by Stevenlee731 — COMMENTED** [review]
Removed the extra comments!

**Pull request merged**